diff --git a/includes/install.inc b/includes/install.inc
index 5e1d3c6326..e9c7b196d8 100644
--- a/includes/install.inc
+++ b/includes/install.inc
@@ -1299,9 +1299,14 @@ function install_profile_info($profile, $locale = 'en') {
       'php' => DRUPAL_MINIMUM_PHP,
     );
     $info = drupal_parse_info_file("profiles/$profile/$profile.info") + $defaults;
+    $profile_dependencies = array();
+    foreach ($info['dependencies'] as $dependency) {
+      $dependency_data = drupal_parse_dependency($dependency);
+      $profile_dependencies[] = $dependency_data['name'];
+    }
     $info['dependencies'] = array_unique(array_merge(
       drupal_required_modules(),
-      $info['dependencies'],
+      $profile_dependencies,
       ($locale != 'en' && !empty($locale) ? array('locale') : array()))
     );
 
