diff --git a/profiles/openideal/modules/contrib/watcher/js/watcher.js b/profiles/openideal/modules/contrib/watcher/js/watcher.js
index d3550dd..93df70d 100644
--- a/profiles/openideal/modules/contrib/watcher/js/watcher.js
+++ b/profiles/openideal/modules/contrib/watcher/js/watcher.js
@@ -3,7 +3,7 @@
  * 
  * By Jakob Persson of NodeOne
  */
-
+(function($) {
 /**
  * Register Watcher behavior
  */
@@ -166,4 +166,6 @@ function toggleWatch(el, rdata) {
   
   // Fade in, display the element again
   $(el).fadeTo(400, 1);
-}
\ No newline at end of file
+}
+
+})(jQuery);
\ No newline at end of file
